wolfssl/wolfcrypt/settings.h

```diff
@@ -172,6 +172,9 @@
 /* Uncomment next line if building for using XILINX */
 /* #define WOLFSSL_XILINX */
 
+/* Uncomment next line if building for  WICED Studio. */
```

Comment:
Can you remove the extra space after "for" in "building for  WICED Studio"?  Thanks!